test/src/main/java/org/apache/accumulo/test/ComprehensiveIT.java

```diff
@@ -133,6 +152,61 @@ public static void teardown() {
     SharedMiniClusterBase.stopMiniCluster();
   }
 
+  @Test
+  public void testEventualScan() throws Exception {
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
```

Comment:
Eventual scans may fall back to tablet servers if no scan servers are present.  This code starts the scans server and then continues the test, so the scan server may still be starting up while the test is running.  To avoid this we could update the client config using the changes from #4671 to avoid falling back to a tablet server.

```suggestion
    Properties props = new Properties();
    props.putAll(getClientProps());
    // TODO configure scan server selector to wait for scan servers by adding to props
    try (AccumuloClient client = Accumulo.newClient().from(props).build()) {
```

Reply:
I opened #4814 to change the default behavior to no fall back.  If that change is merged in then no changes would be needed in this test.